src/Hooks/useCheckAppVersion/useCheckAppVersion.tsx

```diff
@@ -0,0 +1,112 @@
+import moment from "moment"
+import { useEffect, useMemo } from "react"
+import DeviceInfo from "react-native-device-info"
+import { useQuery } from "@tanstack/react-query"
+import { selectUpdatePromptStatus, useAppDispatch, useAppSelector, VersionUpdateSlice } from "~Storage/Redux"
+import { PlatformUtils } from "~Utils"
+import SemanticVersionUtils from "~Utils/SemanticVersionUtils"
+import { VersionManifest } from "~Model/AppVersion"
+
+const TWENTY_FOUR_HOURS = 1000 * 60 * 60 * 24
+
+const UPGRADE_PROMPT_DELAY_DAYS = {
+    FIRST_PROMPT: 5,
+    SECOND_PROMPT: 10,
+    THIRD_PROMPT: 20,
+}
+
+const VERSION_INFO_URL = "https://versioninfo.dev.veworld.vechain.org/releases"
```

Comment:
I believe this should be a env variable, since on Production we should target a different URL